server/protocols/jmap-integration-testing/cassandra-jmap-integration-testing/src/test/java/org/apache/james/jmap/cassandra/cucumber/CassandraStepdefs.java

```diff
@@ -53,8 +59,16 @@ public void init() throws Exception {
 
     @After
     public void tearDown() {
-        mainStepdefs.tearDown();
-        embeddedElasticSearch.after();
-        temporaryFolder.delete();
+        tearDown(() -> mainStepdefs.tearDown());
+        tearDown(() -> embeddedElasticSearch.after());
+        tearDown(() -> temporaryFolder.delete());
```

Comment:
it's a kind of tearDownQuietly, it prevents going out of tearDown because of an exception on first steps.
I like this idea but the name is not clear enough, and you should use ellipsis to chain lambdas